ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/DistributingExecutor.java

```diff
@@ -43,20 +48,24 @@
  * {@code distributedCmdResponseTimeout}.
  */
 public class DistributingExecutor {
+  private final KsqlConfig ksqlConfig;
   private final CommandQueue commandQueue;
   private final Duration distributedCmdResponseTimeout;
   private final BiFunction<KsqlExecutionContext, ServiceContext, Injector> injectorFactory;
   private final Optional<KsqlAuthorizationValidator> authorizationValidator;
   private final ValidatedCommandFactory validatedCommandFactory;
   private final CommandIdAssigner commandIdAssigner;
+  private final ReservedInternalTopics internalTopics;
 
   public DistributingExecutor(
+      final KsqlConfig ksqlConfig,
```

Comment:
I'd actually pass in `ReservedInternalTopics` instead, otherwise you'll need to also pass in `ProcessingLogConfig` too.